### PR TITLE
Add spatial filter to data endpoint with tiles and level

### PR DESCRIPTION
**Pull request by vdubr**

### CODE REVIEW

File: src/components/maps/Map.js [hunk 1]
```diff
@@ -42,6 +46,10 @@ const mapDispatchToPropsFactory = () => {
                     dispatch(Action.maps.updateMapAndSetView(ownProps.stateMapKey, update));
                 },
 
+                onPropViewChange: (update, mapWidth, mapHeight) => {
+                    dispatch(Action.maps.use(ownProps.stateMapKey, undefined, undefined, mapWidth, mapHeight));
```
Comment (vlach1989) [inline src/components/maps/Map.js:50]: we usually use null in case the parameter is missing

File: src/state/Maps/actions.js [hunk 2]
```diff
@@ -15,14 +15,20 @@ const {actionGeneralError} = commonActions;
...
-function use(mapKey, backgroundLayer, layers, spatialFilter) {
+function use(mapKey, backgroundLayer, layers, mapWidth, mapHeight) {
     return (dispatch, getState) => {
         // TODO clear use for given mapKey, if exists
         const state = getState();
         const componentId = `map_${mapKey}`;
         const activeKeys = commonSelectors.getAllActiveKeys(state);
-
+        const spatialFilter = {};
+        if(mapWidth, mapHeight) {
```
Comment (vlach1989) [inline src/state/Maps/actions.js:25]:
```suggestion
        if(mapWidth && mapHeight) {
```

Comment (vdubr) [reply src/state/Maps/actions.js:25]: How it could work???

Comment (vdubr) [reply src/state/Maps/actions.js:25]: thanks

**Pull request merged**